storage/innobase/lock/lock0lock.cc

```diff
@@ -513,6 +510,22 @@ request is really for a 'gap' type lock */
   ut_ad(lock_get_type_low(lock2) == LOCK_REC);
 
   const bool is_hp = trx_is_high_priority(trx);
+
+#ifdef WITH_WSREP
+  if (is_hp && trx != lock2->trx && trx_is_high_priority(lock2->trx)) {
+    /* conflicting high priority locks are supposed to be false positives
+       i.e. locks not on actual certified pay load data or GAP locks.
+       these should not block applier execution, hence returning false here */
+    if (wsrep_debug)
+      ib::info() << "BF-BF lock conflict skipped, \n"
+                 << " trx-1 lock request: " << trx->id
+                 << " trx-2 lock holder: " << lock2->trx->id;
```

Comment:
I think we can print some lock info as well.  there is lock_print() or something like that. 
Would be good to know for debugging purposes.
Also maybe the THD id, other WSREP info

Reply:
done